{{cookiecutter.project_shortname}}/Dockerfile

```diff
@@ -20,12 +20,25 @@
 
 FROM inveniosoftware/centos7-python:3.6
 
+ARG include_assets
+
 COPY Pipfile Pipfile.lock ./
 RUN pipenv install --deploy --system
 
+COPY ./ .
 COPY ./docker/uwsgi/ ${INVENIO_INSTANCE_PATH}
 COPY ./invenio.cfg ${INVENIO_INSTANCE_PATH}
 COPY ./templates/ ${INVENIO_INSTANCE_PATH}/templates/
-COPY ./ .
+
+RUN if [ "$include_assets" = "true" ]; \
+    then \
+        cp -r ./static/. ${INVENIO_INSTANCE_PATH}/static/ && \
+        cp -r ./assets/. ${INVENIO_INSTANCE_PATH}/assets/ && \
+        invenio collect --verbose  && \
+        invenio webpack create && \
+        # --unsafe needed because we are running as root
+        invenio webpack install --unsafe && \
+        invenio webpack build \
```

Comment:
minor: We can use `webpack buildall` here right?

Reply:
indeed